yaksh/views.py

```diff
@@ -2348,6 +2353,14 @@ def show_lesson(request, lesson_id, module_id, course_id):
     learn_module = course.learning_module.get(id=module_id)
     learn_unit = learn_module.learning_unit.get(lesson_id=lesson_id)
     learning_units = learn_module.get_learning_units()
+
+    # if learning module is active
```

Comment:
if learning module is not active.
You an omit the comment if you want because code indicates the intention here.